collector/system_collector.go

```diff
@@ -49,6 +68,60 @@ func newSystemCollector(systemClient client.SystemClient, logger log.Logger) *sy
 		[]string{"ip_address", "type"},
 		nil,
 	)
+	clusterNodeCPUCores := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "cluster_node", "cpu_cores"),
```

Comment:
Do we need metrics about number of CPU cores?

Reply:
this is equivalent to `memory_total`, but unfortunately cpu_util is not exposed, let's just delete then?

Reply:
update: load average can be used for cpu util